gstd/src/prog/generator.rs

```diff
@@ -0,0 +1,63 @@
+// This file is part of Gear.
+
+// Copyright (C) 2022 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! Program generation module
+
+use crate::{common::errors::Result, prog, ActorId, CodeHash};
+use codec::alloc::vec::Vec;
+
+/// ProgramGenerator allows you to create programs without having to set the
```

Comment:
Ticks are ok. E.g.: https://doc.rust-lang.org/src/alloc/string.rs.html#360
```suggestion
/// `ProgramGenerator` allows you to create programs without having to set the
```

Reply:
done